lib/orderbook/OrderBook.ts

```diff
@@ -1,26 +1,36 @@
 import uuidv1 from 'uuid/v1';
 
-import OrderBookRepository, { Orders } from './OrderBookRepository';
+import OrderBookRepository from './OrderBookRepository';
 import MatchingEngine from './MatchingEngine';
 import MatchesProcessor from './MatchesProcessor';
 import errors from './errors';
 import Pool from '../p2p/Pool';
 import { orders, matchingEngine, db } from '../types';
 import DB from '../db/DB';
-import { groupBy } from '../utils/utils';
 import Logger from '../Logger';
 import LndClient from '../lndclient/LndClient';
+import { StampedOrder } from '../types/orders';
+import { ms } from '../utils/utils';
 
 type OrderBookConfig = {
   internalmatching: boolean;
 };
 
+type Orders = {
+  buyOrders: StampedOrder[];
+  sellOrders: StampedOrder[];
+};
+
 class OrderBook {
+  public pairs: db.PairInstance[] = [];
+  public matchingEngines: { [ pairId: string ]: MatchingEngine } = {};
+
   private logger: Logger = Logger.orderbook;
   private repository: OrderBookRepository;
   private matchesProcessor: MatchesProcessor = new MatchesProcessor();
-  public pairs: db.PairInstance[] = [];
-  public matchingEngines: {[ pairId: string ]: MatchingEngine} = {};
+
+  private ownOrders: { [ pairId: string ]: Orders } = {};
+  private peerOrders: { [ pairId: string ]: Orders } = {};
```

Comment:
We don't have to do it now,  but these lists will need to support updates/deletes by the keys: `orderId`, `hostId`. 
So we better remember to optimize it to avoid having to do a "full table scan" (at least for `orderId`) on each operation.